tests/cpp/test_resize.cpp

```diff
@@ -1900,6 +1902,9 @@ TEST_F(ResizeTest, FusionSliceForNanoGPT2) {
 
 // C++ version of TestNvFuserFrontend.test_nanogpt_split_mha_linears
 TEST_F(ResizeTest, FusionSliceForNanoGPT3) {
+  preseg_passes::OptimizationPassGuard<preseg_passes::MarkAliasesPreparePass>
+      optimization_guard(false);
```

Comment:
naive question, why are we skipping alias pass on this?

Reply:
I'll add a comment to explain.

This test is checking 
```
NVF_CHECK(
      !kernel->summary().has_cooperative_grid_reduction,
      "Grid sync should not be used as slicing input should avoid input caching");
```
If we don't skip aliasing, the compilation will be skipped, and this property cannot be verified.
So I am skipping this pass, so the test can actually verify this behavior.

Reply:
We could also modify the test itself, to have other operators to avoid the outputs being aliased.